packages/block-editor/src/components/block-preview/index.js

```diff
@@ -29,10 +32,14 @@ export function BlockPreview( {
 	}
 	return (
 		<BlockEditorProvider value={ renderedBlocks } settings={ settings }>
-			<AutoHeightBlockPreview
-				viewportWidth={ viewportWidth }
-				__experimentalPadding={ __experimentalPadding }
-			/>
+			{ __experimentalLive ? (
+				<LiveBlockPreview onClick={ __experimentalOnClick } />
```

Comment:
I  wonder if the onClick prop really needs to be supported in the preview component itself or more added to a wrapper "button" like we do for patterns.

Reply:
It is added to a wrapper. `LiveBlockPreview` is an internal component.

Reply:
I was thinking more  something like this:

```
<div role="button" onClick={doSomething}>
   <BlockPreview>
</div>
```

Reply:
That could work, but we lose the ability for `BlockPreview` to hook into it. I wonder if that would be useful. If by the time we have to remove the experimental prefix, a use-case hasn't arisen, then I guess we can move it outside.